### DISCUSSION TITLE
CubeFS using Minio components

### DISCUSSION THREAD

**Discussion by vishnu-chalil** [cubefs/cubefs#2743] [General]
Hi, while going through the cubeFS notice in github, I saw that the cubeFS uses the source code from Minio. Minio  a few years back changed the license to GPL3.0. Does this project effect due to that policy change?

**Comment by leonrayang** [comment]
@vishnu-chalil In fact, we have not been using any code related to Minio since version 3.3.0. However, the references and explanations related to it have not been removed. We plan to remove them in version 3.3.2, which is expected to be released in February.
